src/Util/Tokens.php

```diff
@@ -154,6 +154,10 @@
     define('T_ATTRIBUTE', 'PHPCS_T_ATTRIBUTE');
 }
 
+if (defined('T_ENUM') === false) {
+    define('T_ENUM', 'PHPCS_T_ENUM');
+}
+
```

Comment:
This needs to be moved to below the `8.1 tokens` marker.

Reply:
Fixed.